Frameworks/AddressBook/ABContactInternal.mm

```diff
@@ -15,14 +15,17 @@
 //******************************************************************************
 
 #import "ABContactInternal.h"
+#import "ABAddressBookManagerInternal.h"
 #import "UWP/WindowsApplicationModelContacts.h"
 
 @implementation _ABContact
 
-- (id)initWithContact:(WACContact*)contact {
+- (id)initWithContact:(WACContact*)contact andType:(ABRecordContactType)type {
     self = [super init];
     if (self) {
         self.contact = contact;
+        self.manager = nil;
```

Comment:
Don't call messages on a currently initializing object. Use the ivars directly to avoid bad messages. Setters and getters should not be used in init and dealloc methods.